package.json

```diff
@@ -2,11 +2,11 @@
   "name": "@vectara/react-search",
   "version": "0.0.9",
   "description": "A Vectara-powered Search component",
-  "main": "dist/index.cjs.js",
-  "module": "dist/index.esm.js",
-  "types": "dist/index.d.ts",
+  "main": "lib/index.cjs.js",
+  "module": "lib/index.esm.js",
+  "types": "lib/index.d.ts",
   "scripts": {
-    "build": "npm run clean && node build.js && tsc --emitDeclarationOnly --outDir dist",
+    "build": "npm run clean && node build.js && tsc --emitDeclarationOnly --outDir lib",
     "buildDocs": "node docs/build.js",
     "docs": "node docs/docsServer.js",
     "clean": "rimraf dist",
```

Comment:
Should this be `rimraf lib`? If this script isn't being used we could just remove it.

Reply:
Great catch!